lib/scss_lint/linter/space_around_operator.rb

```diff
@@ -0,0 +1,57 @@
+module SCSSLint
+  # Checks for space around operators on values.
+  class Linter::SpaceAroundOperator < Linter
+    include LinterRegistry
+
+    def visit_script_operation(node)
+      source = source_from_range(node.source_range)
+      left_range = node.operand1.source_range
+      right_range = node.operand2.source_range
+
+      # We need to #chop at the end because an operation's operand1 _always_
+      # includes one character past the actual operand (which is either a
+      # whitespace character, or the first character of the operation).
+      left_source = source_from_range(left_range).chop
+      right_source = source_from_range(right_range)
```

Comment:
I added `chop` instead, which is working in all the examples I have (removing semicolons, right parens, left curly braces...).